src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/hmi/rc_on_remote_control_settings_notification.cc

```diff
@@ -0,0 +1,127 @@
+#include "rc_rpc_plugin/commands/hmi/rc_on_remote_control_settings_notification.h"
+#include "rc_rpc_plugin/rc_rpc_plugin.h"
+#include "rc_rpc_plugin/rc_module_constants.h"
+#include "utils/macro.h"
+
+namespace rc_rpc_plugin {
+namespace commands {
+
+namespace {
+std::map<std::string, hmi_apis::Common_RCAccessMode::eType> access_modes{
+    {enums_value::kAutoAllow, hmi_apis::Common_RCAccessMode::AUTO_ALLOW},
+    {enums_value::kAutoDeny, hmi_apis::Common_RCAccessMode::AUTO_DENY},
+    {enums_value::kAskDriver, hmi_apis::Common_RCAccessMode::ASK_DRIVER}};
+}
+
+CREATE_LOGGERPTR_GLOBAL(logger_, "RemoteControlModule")
+
+RCOnRemoteControlSettingsNotification::RCOnRemoteControlSettingsNotification(
+    const app_mngr::commands::MessageSharedPtr& message,
+    app_mngr::ApplicationManager& application_manager,
+    app_mngr::rpc_service::RPCService& rpc_service,
+    app_mngr::HMICapabilities& hmi_capabilities,
+    policy::PolicyHandlerInterface& policy_handle,
+    ResourceAllocationManager& resource_allocation_manager)
+    : application_manager::commands::NotificationFromHMI(message,
+                                                         application_manager,
+                                                         rpc_service,
+                                                         hmi_capabilities,
+                                                         policy_handle)
+    , resource_allocation_manager_(resource_allocation_manager) {}
+
+RCOnRemoteControlSettingsNotification::
+    ~RCOnRemoteControlSettingsNotification() {}
+
+hmi_apis::Common_RCAccessMode::eType AccessModeFromString(
+    const std::string& access_mode) {
+  std::map<std::string, hmi_apis::Common_RCAccessMode::eType>::const_iterator
+      mode = access_modes.find(access_mode);
+  return access_modes.end() != mode
+             ? mode->second
+             : hmi_apis::Common_RCAccessMode::INVALID_ENUM;
+}
+
+std::string AccessModeToString(
+    const hmi_apis::Common_RCAccessMode::eType access_mode) {
+  std::map<std::string, hmi_apis::Common_RCAccessMode::eType>::const_iterator
+      it = access_modes.begin();
+  for (; access_modes.end() != it; ++it) {
+    if (access_mode == it->second) {
+      return it->first;
+    }
+  }
+  const std::string error = "UNKNOW_ACCESS_MODE";
+  DCHECK_OR_RETURN(false, error);
+  return error;
+}
+
+void UnsubscribeFromInteriorVehicleDataForAllModules(
+    RCAppExtensionPtr extension) {
+  LOG4CXX_AUTO_TRACE(logger_);
```

Comment:
Unsubscribe From InteriorVehicleData For All Modules
it is for ALL modules, why I only see 2 of them?